package.json

```diff
@@ -75,6 +75,9 @@
     "validate-commit-msg": "^2.6.1",
     "webpack": "^2.7.0"
   },
+  "overrides": {
+    "graceful-fs": "^4.2.11"
```

Comment:
**Why this change?**

The reason that we were locking node to v10 in this project was because of a specific version of this `graceful-fs` package. Check [this](https://github.com/tractionguest/cordova-plugin-qrscanner/pull/3/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R4)

**The problem**
- We have **gulp@3** which imports **graceful-fs@3**. Now, `graceful-fs@3` overrides some utility from `fs` package which stops working beyond **node@12**. The reason why we kept node at v10.

**The solution**
- The options were either to upgrade gulp to v4 which would mean rewriting gulpfile configuration OR override resolution for `graceful-fs` to v4
- I went with the latter one and verified it to be working fine.

Read more about this issue [here](https://timonweb.com/javascript/how-to-fix-referenceerror-primordials-is-not-defined-error/)